decidim-admin/spec/shared/manage_process_steps_examples.rb

```diff
@@ -97,7 +98,7 @@
     let!(:process_step2) { create(:participatory_process_step, participatory_process: participatory_process) }
 
     before do
-      visit decidim_admin.participatory_process_path(participatory_process)
+      visit current_path
```

Comment:
> I need to do this reload in order to show the newly created record.

First I create a record and visit a path, on top of the file. Then, in this single context, I need to create another record and visualize it. Since I'm already in the page before creating the second record, I need to reload the page in order to see the new record.

I could not find any `reload_page` or similar, so I used a `visit current_path` :/